chainstate/src/detail/transaction_verifier/mod.rs

```diff
@@ -87,11 +100,12 @@ impl<'a, S> TransactionVerifier<'a, S> {
             tx_index_cache: BTreeMap::new(),
             utxo_cache,
             utxo_block_undo: BTreeMap::new(),
+            tokens_cache: BTreeMap::new(),
         }
     }
 }
 
-impl<'a, S: BlockchainStorageRead> TransactionVerifier<'a, S> {
+impl<'a, S: BlockchainStorageRead + 'a> TransactionVerifier<'a, S> {
```

Comment:
Why?

Reply:
Due to last merges, and code simplifying, it's not necessary again now. Fixed.